state/utils/utils.go

```diff
@@ -13,6 +13,26 @@ limitations under the License.
 
 package utils
 
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+func JSONStringify(value any) ([]byte, error) {
+	switch value := value.(type) {
+	case []byte:
+		return value, nil
+	case int, int8, int16, int32, int64, uint, uint8, uint16, uint32, uint64, float32, float64, bool:
+		return []byte(fmt.Sprintf("%v", value)), nil
+	case string:
+		return []byte(`"` + strings.ReplaceAll(value, `"`, `\"`) + `"`), nil
+	default:
+		return json.Marshal(value)
+	}
+}
+
+// Deprecated: use JSONStringify instead.
```

Comment:
Should this be deleted then?

Reply:
It is still references in `dapr/dapr`. We can remove once we have merged this, updated go mod and migrated the caller in `dapr/dapr`